src/auth/auth.service.ts

```diff
@@ -25,24 +25,57 @@ export class AuthService {
     private readonly jwtService: JwtService,
   ) {}
 
-  async verifyJwt(jwtToken: string): Promise<AccessTokenPayload> {
+  verifyJwt(jwtToken: string): Promise<AccessTokenPayload> {
     return this.jwtService.verifyAsync(jwtToken);
   }
 
-  async signIn(signInRequest: SignInRequest): Promise<string> {
-    const user = await this.userRepository.getUserByEmailWithPassword(
-      signInRequest.email,
+  async generateBearerToken(
+    id: number,
+    secret: string,
+    timeInSeconds?: number,
+  ): Promise<string> {
+    const payload = { id };
+    const options: JwtSignOptions = { secret };
+
+    if (timeInSeconds) {
+      options.expiresIn = `${timeInSeconds}s`;
```

Comment:
hmm, perhaps I wouldn't hardcode seconds here, but just pass a value from environment value that would be either `60s` or `2h` or `7d` or something else